lib/advisor/core/answers/answers.ex

```diff
@@ -1,13 +1,30 @@
-defmodule Advisor.Core.AnswerFinder do
-  alias Advisor.Core.Answer
+defmodule Advisor.Core.Answers do
   alias Advisor.Repo
+  alias Advisor.Core.Answer
   import Ecto.Query
 
+  def store(params) do
+    Repo.insert_all(Answer, all_answers(params), returning: true)
+  end
+
+  def all_answers(%{"id" => advice_request_id} = params) do
+    Enum.reduce(params, [], fn({question_id, answer}, acc) ->
```

Comment:
@fabientownsend Stands for "accumulator" which is a common name in FP when using the `reduce` function. You _accumulate_ a result. I often try to rename it to what the actual end result is. Another thing that could help in terms of readability/understanding is extracting the function inside the `reduce`.